R/add_resource.R

```diff
@@ -32,21 +40,25 @@
 #'   y = c(860, 900)
 #' )
 #'
-#' # Add the data frame as a new resource to the Data Package
-#' package <- add_resource(package, "positions", df)
+#' # Add a new resource "positions" to the Data Package, from the data frame
+#' package <- add_resource(package, "positions", data = df)
 #'
-#' # List the resource names ("positions" added)
+#' # Add a new resource "deployments2" to the Data Package, from a CSV file path
```

Comment:
I think it would be worth to add the following examples:
1. multiple paths
2. user passes a schema via arg `schema`